.web-docs/components/builder/vsphere-clone/README.md

```diff
@@ -749,6 +749,8 @@ wget http://{{ .HTTPIP }}:{{ .HTTPPort }}/foo/bar/preseed.cfg
 
 - `vTPM` (bool) - Add virtual TPM device for virtual machine. Defaults to `false`.
 
+- `VirtualPrecisionClock` (string) - Add precision clock device for virtual machine. Defaults to `none`.
```

Comment:
Would prefer either `precision_clock` or `precision_clock` as it it referred to in vSphere (docs and UI...)

```suggestion
- `precision_clock` (string) - Add a precision clock device for virtual machine. Defaults to `none`.
```